### PR TITLE
itemfix.com cyprus-mail.com realmicentral.com filetransfer.io

### PR DESCRIPTION
**Pull request by FadeMind**
https://www.itemfix.com/?r=ll

### CODE REVIEW

**Comment by FadeMind** [comment]
https://cyprus-mail.com

**Comment by FadeMind** [comment]
https://www.realmicentral.com/

**Comment by FadeMind** [comment]
https://komorkomat.pl/na-karte/

**Comment by FadeMind** [comment]
https://filetransfer.io/

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -8689,6 +8713,15 @@ IGNORE IMAGE ANALYSIS
 
 ================================
 
+realmicentral.com
+
+INVERT
+img[itemprop="logo"]
+div[class="fly-but-wrap left relative"] > span
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:8720]:
```suggestion
div.fly-but-wrap.left.relative > span
```

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -8689,6 +8713,15 @@ IGNORE IMAGE ANALYSIS
...
+div[class="fly-but-wrap left relative fly-open"] > span
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:8721]: Remove this one as the above one will take into account this one.

**Review by Gusted — APPROVED** [review]

**Pull request merged**